### PR TITLE
gh-100488: Add is_integer method to fractions.Fraction

### PR DESCRIPTION
**Pull request by hauntsaninja**
* Issue: gh-100488

### CODE REVIEW

**Comment by hauntsaninja** [comment]
The documentation guarantees that denominator is in lowest term. It doesn't make any explicit guarantee about being positive, but `as_integer_ratio` does and just returns the denominator. But do let me know if I should change to `abs(self._denominator) == 1` or `self._numerator % self._denominator == 0`!

**Review by JelleZijlstra — APPROVED** [review]
(But let's wait for Mark to also give feedback.)

File: Doc/library/fractions.rst [hunk 1]
```diff
@@ -117,6 +117,12 @@ another rational number, or from a string.
 
       .. versionadded:: 3.8
 
+   .. method:: is_integer()
+
+      Return True if the Fraction is an integer.
```
Comment (mdickinson) [inline Doc/library/fractions.rst]: Nitpick: suggest inline code markup for `True`:
```suggestion
      Return ``True`` if the Fraction is an integer.
```

(We seem to be rather inconsistent about this across the codebase. The most authoritative source I could find is in the dev guide: https://devguide.python.org/documentation/markup/#quick-reference.)

**Review by mdickinson — APPROVED** [review]
LGTM; one reST markup nitpick.

**Pull request merged**